packages/infrastructure/kube_airbyte/main.tf

```diff
@@ -0,0 +1,1318 @@
+            memory = "${var.workload_api_server_memory_request_mb * var.memory_limit_multiplier}Mi"
+          }
+        }
+      }
+
+      # Workload launcher configuration
+      "workload-launcher" = {
+        enabled        = true
+        podLabels      = module.util_workload_launcher.labels
+        podAnnotations = var.pod_annotations
+
+        affinity     = module.util_workload_launcher.affinity
+        tolerations  = module.util_workload_launcher.tolerations
+        nodeSelector = var.node_selector
+
+        resources = {
+          requests = {
+            memory = "${var.workload_launcher_memory_request_mb}Mi"
+            cpu    = "${var.workload_launcher_cpu_request_millicores}m"
+          }
+          limits = {
+            memory = "${var.workload_launcher_memory_request_mb * var.memory_limit_multiplier}Mi"
+          }
+        }
+      }
+
+      # Bootloader configuration
+      "airbyte-bootloader" = {
+        env_vars = {}
+      }
+    })
+  ]
+
+  depends_on = [
+    module.database,
+    kubernetes_secret.airbyte_secrets,
+    module.aws_permissions
+  ]
+}
+
+/***************************************
+* Airbyte Ingress
+***************************************/
+
+module "authenticating_proxy" {
+  count  = var.ingress_enabled ? 1 : 0
+  source = "../kube_vault_proxy"
+
+  namespace                            = local.namespace
+  spot_nodes_enabled                   = var.spot_nodes_enabled
+  burstable_nodes_enabled              = var.burstable_nodes_enabled
+  controller_nodes_enabled             = var.controller_nodes_enabled
+  pull_through_cache_enabled           = var.pull_through_cache_enabled
+  vpa_enabled                          = var.vpa_enabled
+  domain                               = var.domain
+  vault_domain                         = var.vault_domain
+  instance_type_anti_affinity_required = var.sla_target == 3
+  az_spread_preferred                  = var.sla_target >= 2
+  panfactum_scheduler_enabled          = var.panfactum_scheduler_enabled
+  wait                                 = var.wait
+}
+
+module "ingress" {
+  count     = var.ingress_enabled ? 1 : 0
+  source    = "../kube_ingress"
+  namespace = local.namespace
+  name      = "airbyte"
+  domains   = [var.domain]
+  ingress_configs = [{
+    service      = "${local.name}-airbyte-webapp-svc"
+    service_port = 80
+  }]
+
+  rate_limiting_enabled          = true
+  cross_origin_isolation_enabled = false
+  cross_origin_embedder_policy   = "credentialless"
+  permissions_policy_enabled     = true
+  csp_enabled                    = true
+  cross_origin_opener_policy     = "same-origin-allow-popups"
+
+  # CSP configuration
+  csp_style_src  = "'self' 'unsafe-inline'"
+  csp_script_src = "'self' 'unsafe-inline'"
+  csp_img_src    = "'self' data:"
+
+  extra_annotations = merge(
+    module.authenticating_proxy[0].upstream_ingress_annotations,
+    {
+      "nginx.ingress.kubernetes.io/proxy-connect-timeout" = "600"
```

Comment:
why?

Reply:
when testing out their UI, there are certain api actions like initializing a new connector that took a long time and the UI treatment was to wait for it. Given it also uses websockets, I was suprised this was the case but when the timeout occurs, it didn't resume and started over and became a blocker to allow for a longer timeout period here.